account-and-billing/account-and-billing.constant.ts

```diff
@@ -0,0 +1,51 @@
+export class AccountAndBillingConstant {
+    static readonly attributes = Object.freeze({
+        text: Object.freeze({
+            accountAndBilling: 'Account and Billing',
+            pastPayments: 'Past Payments',
+            accountId: 'Account ID',
+            customerSupport: 'Customer Support',
+            primaryContact: 'Primary Contact',
+            exportData: 'Export Data',
+            downloadLink: 'Download FogBugz data now',
+            closeAccount: 'Close Account',
+            closedAccountsCannotBeRecovered: 'Closed accounts cannot be recovered',
+            editProducts: 'Edit Products',
+            paymentPlans: 'Payment Plans',
+            addCreditCard: 'Add Credit Card',
+            creditCard: 'Credit Card',
+            nextBill: 'Next Bill',
+            contactCustomerSupport: 'Contact Customer Support',
+            supportUrl: 'https://support.fogbugz.com/hc/en-us',
+            blank: '_blank',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400120)
```suggestion
            blank: "_blank",
```